db/migrations/18_do_not_mark_ancestors_as_todo_in_groups_groups_triggers.sql

```diff
@@ -1,12 +1,12 @@
 -- +migrate Up
 DROP TRIGGER `before_update_groups_groups`;
 -- +migrate StatementBegin
-/*!50003 CREATE*/ /*!50017 DEFINER=`algorea`@`%`*/ /*!50003 TRIGGER `before_update_groups_groups`
+/*!50003 CREATE*/ /*!50003 TRIGGER `before_update_groups_groups`
```

Comment:
Without `/*!50017` comment, probably it is useless to close the 50003 comment to reopen it directly, so it should probably be `/*!50003 CREATE TRIGGER ...`
BTW, I think all these comments are irrelevant in the case of our migrations. In my understanding, they are there to prevent error if we would apply this to a version of MySQL not supporting these, but in our case, we would prefer an error than ignoring the trigger silently. 
I think all comment marks should all be removed in our migrations (and probably in the initial schema as well)